changelog

```diff
@@ -1,3 +1,66 @@
+Version 0.10.10
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@alextd, @opusforlife2, @TheGiraffe3, @tibetiroka)
+      * Added "required crew" to Successors' turrets. (@tibetiroka)
+      * Corrected offer and fail conditions of "Successors: First Contact 2 Prompt". (@Daeridanni1)
+      * Added clearance to "Successors: Predecessor Research 1". (@TheGiraffe3)
+      * "First Contact: Quarg" and "Ringworld Debris: Quarg" will now only offer on ringworlds in human and Hai space. (@tibetiroka, @TheGiraffe3)
+      * Added missing government to uninhabited systems near the Incipias. (@mOctave)
+      * Removed a spurious condition operation in "Hogshead's Dilemma II". (@TheGiraffe3)
+      * Successors rush delivery jobs will no longer choose the origin, or another planet less than two jumps away, as the destination. (@reticent-rem)
+      * Added a death explosion "weapon" to the Incipias ships. (@TheGiraffe3, @Quantumshark)
+      * "Successors: Researchers Out 1" and "Successors: Large Researchers Out 1" will now correctly calculate expected jumps for dynamic payments based on transit by wormhole and Jump Drive. (@tekker2234)
+      * Added display names, instead of using the unique names, for various early Wanderer missions. (@TheGiraffe3)
+      * Gave display names and descriptions to some early Incipias missions. (@TheGiraffe3)
+      * When selling a Jump Drive to the Unfettered, they will no longer provide a replacement Hyperdrive if the Jump Drive was in cargo. (@Hurleveur)
+      * Corrected the date on which "Earth Day Blackout" will offer. (@TheGiraffe3)
+      * The missions "Gegno Anticipation" and "Gegno Intervention" will no longer offer after the other has been offered. (@Saugia)
+      * Added missing passengers to "Remnant: Cognizance 7" and "Remnant: Visit to Paradise 1". (@opusforlife2)
+      * Text in "Remnant: Void Sprites 3" will now correctly account for if the player was given additional information in "Remnant: Void Sprites 1". (@warp-core)
+      * "Remnant: Shattered Light 3" will no longer remain active after "Remnant: Shattered Light 4" has started, preventing the presence of two Shatered Lights at the same time. (@warp-core)
+      * Updated some Korath ship variants to use the "Piercer Missile Launcher" instead of the deprecated "Korath Piercer Launcher". (@Zitchas)
```

Comment:
```suggestion
      * Updated some Korath ship variants to use the "Piercer Missile Launcher" instead of the deprecated "Korath Piercer Launcher". (@Zitchas)
      * Updated the same ship variants to use "Piercer Missile" instead of the deprecated "Korath Piercer".
```

Reply:
I'd rather use one entry than have two almost identical lines.